readme.md

```diff
@@ -0,0 +1,20 @@
+# Selenium project to test (Swag labs)[https://www.saucedemo.com/]
+
+## Required to work:
+
+- python3
+- selenium package
+- google chrome driver
```

Comment:
Is that still needed?

Reply:
yes python3 and selenium is crucial to run the tests, while google chrome driver is optional but installing it may prevent drivers error ( I think when chrome browser is installed the driver is installed also but I put it just in case )

Reply:
Yes, Python 3 and Selenium, but Google Chrome drivers aren't needed anymore, which is why you should remove them.